.craft.yml

```diff
@@ -22,18 +22,6 @@ targets:
     source: us.gcr.io/sentryio/sentry
     target: getsentry/sentry
     targetFormat: '{{{target}}}:latest'
-  - id: py2-release
-    name: docker
-    source: us.gcr.io/sentryio/sentry
-    sourceFormat: '{{{source}}}:{{{revision}}}-py2'
-    target: getsentry/sentry
-    targetFormat: '{{{target}}}:{{{version}}}-py2'
-  - id: py2-latest
-    name: docker
-    source: us.gcr.io/sentryio/sentry
-    sourceFormat: '{{{source}}}:{{{revision}}}-py2'
-    target: getsentry/sentry
-    targetFormat: '{{{target}}}:latest-py2'
 
 requireNames:
   - /^sentry-.+-py2.py3-none-any.whl$/
```

Comment:
Does `py2` still belong in this string?

Reply:
I wasn't sure. I got the feeling that it's simply the naming for wheels that are compatible with Python 2 & 3.
I've seen those filenames in the output of pip installations.

Reply:
Yes as we generate universal bundles. We should change that too, good catch.